test/fft_test.rg

```diff
@@ -17,92 +17,420 @@ import "regent"
 local fft = require("fft")
 
 local cmapper = require("test_mapper")
+local format = require("std/format")
 
-local fft1d = fft.generate_fft_interface(int1d, complex64)
-local fft2d = fft.generate_fft_interface(int2d, complex64)
+-- PRINT FUNCTIONS
+
+local function make_print_region_task(title, input)
+  local t = regentlib.newsymbol(title, "t")
+  local i = regentlib.newsymbol(input, "i")
+  local task print_region_task([t], [i])
+  where reads(i) do
+    format.println("{} = [", t)
+    for x in i do
+      format.println("{},", i[x])
+    end
+    format.println("]")
+  end
+  return print_region_task
+end
+
+local print_region_1d_float = make_print_region_task(rawstring, region(ispace(int1d), float))
+local print_region_1d_double = make_print_region_task(rawstring, region(ispace(int1d), double))
+local print_region_1d_complex32 = make_print_region_task(rawstring, region(ispace(int1d), complex32))
+local print_region_1d_complex64 = make_print_region_task(rawstring, region(ispace(int1d), complex64))
+local print_region_2d_complex64 = make_print_region_task(rawstring, region(ispace(int2d), complex64))
+local print_region_3d_double = make_print_region_task(rawstring, region(ispace(int3d), double))
+local print_region_3d_complex64 = make_print_region_task(rawstring, region(ispace(int3d), complex64))
+local print_region_4d_complex64 = make_print_region_task(rawstring, region(ispace(int4d), complex64))
```

Comment:
Can we change `rawstring` to `regentlib.string`? Technically `rawstring` is not safe to pass to a task because the pointer may not be valid across nodes.

Reply:
Edited here: https://github.com/arjunkunna/regent-fft/commit/103303a67e88d24b221b1de468e6d7bf2cc806d5